internal/api/connectors/cloudConnector_test.go

```diff
@@ -263,19 +264,9 @@ var _ = Describe("Cloud Connector", func() {
 
 			client := NewConnectorClientWithHttpRequestDoer(config.Get(), &doer)
 			ctx := utils.SetLog(test.TestContext(), zap.NewNop().Sugar())
-			_, err := client.GetConnectionStatus(ctx, "901578", "5318290", "be175f04-4634-49f2-a292-b4ad7107af78")
+			_, err := client.GetConnectionStatus(ctx, nil, "5318290", "be175f04-4634-49f2-a292-b4ad7107af78")
 			Expect(err).ToNot(HaveOccurred())
 
-			bytes, err := ioutil.ReadAll(doer.Request.Body)
-			Expect(err).ToNot(HaveOccurred())
-			parsedRequest := make(map[string]interface{})
-			err = json.Unmarshal(bytes, &parsedRequest)
-			Expect(err).ToNot(HaveOccurred())
-
-			Expect(parsedRequest["account"]).To(Equal("901578"))
-			Expect(parsedRequest["node_id"]).To(Equal("be175f04-4634-49f2-a292-b4ad7107af78"))
-
-			Expect(doer.Request.Header.Get(constants.HeaderCloudConnectorAccount)).To(Equal("901578"))
```

Comment:
Why are these assertions being removed?

Reply:
The account and node_id are not expected to be in the request doc.  Maybe the test needs to be modified to verify that the psk/header is getting passed along and that the client_id is in the url?